examples/component-composition/app/templates/songs.hbs

```diff
@@ -1,8 +1,8 @@
-<h1>{{songCount}} songs</h1>
+<h1 class="song-count">{{this.songCount}} songs</h1>
 
-<ListView @items={{model}} @onRemoveItem={{action "deleteSong"}} as |list item|>
-  <SongTile @song={{item}} @onSelectSong={{list.selectItem}} />
-  {{#list.delete-button}}
+<ListView @items={{this.model}} @onRemoveItem={{fn this.deleteSong}} as |list item|>
```

Comment:
Is the `fn` needed here?

Reply:
Nice catch 👍 not needed and allows us to remove `@action` as well